dashboard/comparison_taxids.txt

```diff
@@ -346,7 +346,6 @@
 149539
 150247
 153135
-154784
```

Comment:
This is suspicious, but may not be worth digging into?

Reply:
I agree.

Reply:
@jeffkaufman Should I create an issue for future reference or leave it?